src/Explorer/Panes/AddCollectionPanel.tsx

```diff
@@ -122,9 +122,33 @@ export class AddCollectionPanel extends React.Component<AddCollectionPanelProps,
       errorMessage: "",
       showErrorDetails: false,
       isExecuting: false,
-    };
+    }
+
+    if (userContext.features.partitionKeyDefault) {
+      this.state = {
+        createNewDatabase: userContext.apiType !== "Tables" && !this.props.databaseId,
+        newDatabaseId: "",
+        isSharedThroughputChecked: this.getSharedThroughputDefault(),
+        selectedDatabaseId:
+          userContext.apiType === "Tables" ? CollectionCreation.TablesAPIDefaultDatabase : this.props.databaseId,
+        collectionId: "",
+        enableIndexing: true,
+        isSharded: userContext.apiType !== "Tables",
+        partitionKey: "",
+        enableDedicatedThroughput: false,
+        createMongoWildCardIndex: isCapabilityEnabled("EnableMongo"),
+        useHashV2: false,
+        enableAnalyticalStore: false,
+        uniqueKeys: [],
+        errorMessage: "",
+        showErrorDetails: false,
+        isExecuting: false,
+      }
+    }
```

Comment:
You probably don't need to replace this entire object. You can just set the specific property you want to change:

`this.state. partitionKey = "something"`